src/vs/base/browser/ui/inputbox/inputBox.ts

```diff
@@ -402,18 +402,6 @@ export class InputBox extends Widget {
 		const styles = this.stylesForType(this.message.type);
 		this.element.style.border = styles.border ? `1px solid ${styles.border}` : '';
 
-		// ARIA Support
```

Comment:
@anthonydresser Is it preferred to comment this out with a SQL CARBON EDIT here instead of just deleting?

Reply:
commenting out is generally easier to handle for merge conflicts.